gapic/templates/setup.py.j2

```diff
@@ -7,6 +7,8 @@ import setuptools  # type: ignore
 setuptools.setup(
     name='{{ api.naming.warehouse_package_name }}',
     version='0.0.1',
+    author='Google LLC',
+    license='Apache 2.0',
```

Comment:
Not hugely serious, but if we are committed to the generator being used beyond our APIs, should this not be a standalone template that is included and can be overridden more easily? That's what we do for the license header in the generated source files.

Reply:
Ah good points. I didn't realize that's how the license header was getting pulled in. Will close this PR and poke around a bit more.